_source/_posts/2022-04-11-java-cli-device-grant.adoc

```diff
@@ -272,7 +272,7 @@ Assuming all went well, the response returned from the IdP will be a JSON block
   "device_code": "a-random-device-code", # <.>
   "user_code": "ABCD1234", # <.>
   "verification_uri": "https://id.bdemers.io/activate", # <.>
-  "verification_uri_complete": "https:/dev-133337/activate?user_code=ABCD1234", # <.>
+  "verification_uri_complete": "https:/dev-l33337.okta.com/activate?user_code=ABCD1234", # <.>
```

Comment:
It looks like there's a leading L rather than a 1.
```suggestion
  "verification_uri_complete": "https:/dev-133337.okta.com/activate?user_code=ABCD1234", # <.>
```

Reply:
I'm also confused because the `verification_uri` and the `verification_uri_complete` don't share a domain. The description for `verification_uri_complete` seems to imply they should match.

> A URL combining the `verification_uri` and `user_code` which can be shown directly to the user (if they can copy and paste) or rendered as a QR code (on a TV).

Reply:
OH!!! sorry, that's just a copy pasta error, I've been looking at this example for too long now and I didn't even notice it